### PR TITLE
Fix(css): Reactify Free Code Camp

### LINKED ISSUE #9955: [beta] CSS issues with HTML challenges
#### Challenge Name
Any HTML challenge (http://beta.freecodecamp.com/en/challenges/html5-and-css/say-hello-to-html-elements)

#### Browser Information

* Browser Name, Version: 
* Operating System: 
* Mobile, Desktop, or Tablet: 

#### Your Code

```js
 
```

### LINKED ISSUE #9983: [beta] Code editor cannot display full seed code if it is large
No scrollbar here, and I cannot access the end of the code in the editor.

screenshot from 2016-07-29 20 27 26

Browser: Google Chrome latest on Linux Mint.

### LINKED ISSUE #10002: [beta] Map aside looks different than current
#### BETA

image

#### CURRENT

image

### PR DESCRIPTION
**Pull request by raisedadead**
The Instructions area are still narrow but this is the same case as on the current production. Also this addresses the default view port that is a desktop at the moment, we have open issues for mobile ~~and editor scroll~~

Those are to be taken in separate pull requests.

Closes #9955 
Closes #9983
Closes #10002

### CODE REVIEW

**Comment by BerkeleyTrue** [comment]
The padding top is not sufficient:

say-hello-to-html-elements___free_code_camp

**Comment by raisedadead** [comment]
Does this work better?

File: client/less/drawers.less [hunk 1]
```diff
@@ -44,6 +44,17 @@
       width: 100%;
       height: 100%;
     }
+
+    h2 {
+        font-size: 20px;
```
Comment (BerkeleyTrue) [inline client/less/drawers.less]: The indenting here should be two spaces

File: client/less/drawers.less [hunk 2]
```diff
@@ -44,6 +44,22 @@
       width: 100%;
       height: 100%;
     }
+
+    h2 > a {
+        font-size: 20px;
```
Comment (BerkeleyTrue) [inline client/less/drawers.less]: Indent here should be two space

**Comment by raisedadead** [comment]
Okay! Getting closer.

**Comment by raisedadead** [comment]
Done.

**Comment by raisedadead** [comment]
@FreeCodeCamp/issue-moderators please QA.

Note : This should be handling most noticeable CSS issues on desktop. Mobile devices still need to be handled, but that can come later.

I have tried testing as much as I could, do let me know your feedback.

File: client/less/challenge.less [hunk 3]
```diff
@@ -1,6 +1,11 @@
-.challenges-editor {
-  height: 100%;
-  width: 99%;
+/* Challenges instruction Panel*/
```
Comment (BerkeleyTrue) [inline client/less/challenge.less]: These comments are redundant. Please remove them.

**Comment by BerkeleyTrue** [comment]
Other then the unnecessary comments this PR is ready to merge 👍

**Comment by raisedadead** [comment]
Oh yeah, I put them there initially while I was bringing in stuff from various places, will remove and update the pr.

**Pull request merged**